node-notes-cli/writefile.js

```diff
@@ -0,0 +1,11 @@
+const data = require('./data.json');
+const fs = require('fs');
+
+function writeFileReuse() {
```

Comment:
`writeFileReuse` should have  a single parameter of `data` which will be the object you want to save to the file.